### DISCUSSION TITLE
[A/L] (2021/22) Relazioni - Esercizio 11

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#283] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152319278-9270a877-ac6f-4773-97a3-7680ece61fc3.png)

**Comment by LuginiAndrea** [comment]
Riflessività: $aRa \Longrightarrow aR^{-1}a$
Anti-simmetria $aRb \wedge bRa \Longrightarrow b = a$, quindi potremmo sostituire queste due coppie con $aRa$, che vale anche in $aR^{-1}a$. Ne consegue che anche l'inversa è anti-simmetrica in quanto riflessiva
Transitività: $aRb \wedge bRc \Longrightarrow aRc$. Inoltre $bR^{-1}a \wedge cR^{-1}b \wedge cR^{-1}a$. Vale anche qui la transitività
